kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -42,219 +42,9 @@ server {
     # NOTE: OpenResty uses /usr/local/openresty/nginx/, not /etc/nginx/ as the default config location.
     # We have to use absolute path to include files from /etc/nginx/.
     include                  /etc/nginx/snippets/error-pages.conf;
+    include                  /etc/nginx/snippets/file-transfer-endpoints.conf;
 
     set $gatekeeper_api_backend http://gatekeeper.default.svc.cluster.local;
```

Comment:
Does this still work if you set it after the include? Until now it was defined before the endpoints.

Reply:
It's works but I change it to defined before the endpoints.